.github/workflows/soakTest.yml

```diff
@@ -1,7 +1,8 @@
 name: Soak test
 
 on:
-  pull_request: ~
+  # Soak tests are not very reliable at the moment, enable after fixing
```

Comment:
Should we let DF know we're doing this? Thought someone was working on it but in case they merge develop and don't know why they can't test as easily anymore?

Reply:
I've asked @austinborn to take a look at these changes in general

Reply:
@qs is working on soak test reliability right now. I'll make sure he's aware.